Cargo.lock

```diff
@@ -975,8 +963,8 @@ dependencies = [
  "anyhow",
  "backtrace",
  "bellman",
- "blake2b_simd",
- "blake2s_simd 1.0.0",
+ "blake2b_simd 1.0.0",
```

Comment:
How did we end up depending on two versions of `blake2b_simd` when we didn't before?

Reply:
Oh, `equihash` still depends on `blake2b_simd 0.5.11`.